dask/dataframe/shuffle.py

```diff
@@ -114,6 +114,9 @@ def sort_values(
             sort_function, **sort_function_kwargs
         )
 
+    if not isinstance(ascending, bool):
+        raise ValueError("ascending must be either True or False")
```

Comment:
Can you indicate in this message that `ascending` as a list of bools is not supported? Just to communicate to the user that it's a dask limitation, not a mistake that they made

Reply:
Done